sqlserver/datadog_checks/sqlserver/statements.py

```diff
@@ -41,19 +41,19 @@
 
 STATEMENT_METRICS_QUERY = """\
 with qstats as (
-    select text, query_hash, query_plan_hash, last_execution_time,
+    select TOP {limit} text, query_hash, query_plan_hash,
```

Comment:
If we're adding `TOP`, we need to `ORDER BY` something like total_elapsed_time or we'll just get semi-random  set of rows.

Reply:
We agreed to have this be a possible follow-up as it's an extreme edge case where that will matter but there may be a performance impact (albeit a small one) on everyone.